tonic/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "celestia-tonic"
```

Comment:
I think I'd prefer `celestia-grpc` and `grpc` directory

Reply:
good call, done :+1: